ipywidgets/widgets/domwidget.py

```diff
@@ -0,0 +1,76 @@
+"""Contains the DOMWidget class"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from traitlets import Unicode, Dict, Instance, Bool, List, \
+    CaselessStrEnum, Tuple, CUnicode, Int, Set
+from .widget import Widget, widget_serialization
+from .trait_types import Color
+from .style import Style
+
+class DOMWidget(Widget):
+    """Widget that can be inserted into the DOM"""
+    
+    visible = Bool(True, allow_none=True, help="Whether the widget is visible.  False collapses the empty space, while None preserves the empty space.", sync=True)
+    _css = Tuple(sync=True, help="CSS property list: (selector, key, value)")
+    _dom_classes = Tuple(sync=True, help="DOM classes applied to widget.$el.")
+
+    style = Instance(Style, allow_none=True, sync=True, **widget_serialization)
+    def _style_default():
```

Comment:
Would it make more sense for the detail to by None so Python code can test to see if any Style is applied more easily?